pkg/crc/machine/kubeconfig.go

```diff
@@ -227,3 +227,16 @@ func contains(arr []string, str string) bool {
 	}
 	return false
 }
+
+func updateServerDetailsToKubeConfig(input, output string) error {
```

Comment:
I'd keep this one in the commit adding microshift support.

Reply:
This is helper function and used for microshift preset type but it doesn't depend on the preset so created a separate commit.

Reply:
But having `updateServerDetails..` on its own is not really useful, it's an integral part of the microshift commit, and it's a single function in a single file, so it's not making the microshift commit a lot more complicated, which is why I'd keep it there. I can point a few small things which could be moved out of the microshift commit though if you want to make it a bit smaller.